code/modules/mob/living/carbon/metroid/metroid.dm

```diff
@@ -61,14 +61,16 @@
 	var/core_removal_stage = 0 //For removing cores.
 	var/toxloss = 0
 
+var/list/slime_descriptors = list("jiggling","wiggling","bobbing","slippery","moist","slick","translucent","transparent","opaque","sparking","buzzing","crackling","humming","dangerous","friendly","cuddly","malevolent","bouncing","harmless","rare","common","uncommon","ultra-rare","captivating","distrating","hypnotic")

```

Comment:
This is by far the worst way to preform this task. We have a entire document filled with descriptors in the config\names folder, why not just make it select one from this list, this is really really not needed and is just clutter.

You can literally do var/species_descriptors = pick(adjectives) and it will pick from the list which has a large list of usable ones that fit all mobs